string name
float64[] center
float64[] dim
float64[] rot
---
string info
